controllers/userDeviceInfo.js

```diff
@@ -0,0 +1,25 @@
+const UserDeviceInfoModel = require("../models/userDeviceInfo");
+const { SOMETHING_WENT_WRONG } = require("../constants/errorMessages");
+
+const storeUserDeviceInfo = async (req, res) => {
+  try {
+    const userJson = {
+      userId: req.body.userId,
+      deviceType: req.body.deviceType,
```

Comment:
What does deviceType here mean?
What we plan to do is send device info like Manufacturer's name and device name. 
For example: SAMSUNG - Galaxy S20; APPLE - iPhone 6S

Reply:
https://github.com/Real-Dev-Squad/website-data-models/pull/30
Haven't you checked this?